my-app/src/AddBox/AddBox.jsx

```diff
@@ -103,38 +90,44 @@ export default class AddBox extends Component {
   };
 
   handleTagSelection = (tag) => {
-    this.props.addTagToPerson(tag.label);
+    this.props.addTagToPerson(tag.label, this.state.publicity);
   }
 
-  handlePersonSelection = (person) => {
-     this.props.setPerson(person);
+   handleLabelSelection = (label) => {
+    this.props.addTagToPerson(label, this.state.publicity);
   }
 
+
   onSuggestionSelected = (event, { suggestion }) => {
     /**
      *Handle existing selection
     */
     // determine whether suggestion is a person or a tag
-    if (suggestion.hasOwnProperty('label')) {
       // handle selected tag
-      this.handleTagSelection(suggestion);
-    } else if (suggestion.hasOwnProperty('name')) {
-      // handle selected person
-      this.handlePersonSelection(suggestion);
-    }
+    this.handleLabelSelection(suggestion);
     this.setState({value:''});   
   };
   
    _handleKeyPress = e => {
     if (this.state.suggestions.length === 0 && this.state.value !== '') {
           // Handles comlete entries
       if (e.key === 'Enter') {
-        this.props.addThing(this.state.value);
+        console.log("publicity = " + this.state.publicity);
+        this.props.addTagToPerson(this.state.value, this.state.publicity);
         this.setState({value:''});
       }
     }
   };
   
+  onPublicityChanged = e => {   
+    if (this.state.publicity === "private") {
```

Comment:
is there a reason to make it a string rather than a b0oolean?

Reply:
Yes - we'll eventually have more than just private and public - things like 1st, 2nd degree publicity